dist/src/main/resources/bootstrap-cloud.yml

```diff
@@ -0,0 +1,11 @@
+spring:
+  cloud:
+    gcp:
+      core:
+        enabled: true
+      secretmanager:
+        enabled: true
+
+DATABASE_USERNAME: ${sm://reciplease-database-username}
```

Comment:
Maybe we can have best of both worlds: `${RECIPLEASE_DATABASE_USERNAME:sm://reciplease-database-username`?

Allows simulating production on our local machines if needed.

Reply:
local simulation still works! this bootstrap is an extension to the `db-prod` profile - locally you can run `db-prod` with your fancy ENV variables, then in GCP we run `cloud, db-prod`

Reply:
Okay got it. So we'd only run `cloud` when it's running on GCP. That's fine. But for argument sake, I wanted to debug an PROD issue related to GCP. Could I use the profile `cloud` locally and everything will still work?

Regardless, until we've got an issue of that nature. It's not a major issue so I'd consider this point resolved.

Reply:
if you have the ENV variables for db details, you don't need cloud. if you want to run with gcloud features locally, there are ways of doing that, and if we run into issues I can add a profile / make magic to set that up

Reply:
there's an argument that we could remove the use of ENV variables and just use gcloud to store secrets... will require you to install the gcloud CLI